frontend/new-admin-vm/Install-CAM.ps1

```diff
@@ -976,6 +1037,67 @@ graphURL=https\://graph.windows.net/
 				$laSecretVersionedURL = $laSecret.Id
 				$laSecretURL = $laSecretVersionedURL.Substring(0, $laSecretVersionedURL.lastIndexOf('/'))
 
+				################################
+				Write-Host "Populating user blob"
+				################################
+				$container_name = "cloudaccessmanager"
+				$acct_name = $using:userDataStorageAccount
+				$new_agent_vm_files = @(
+					"Install-PCoIPAgent.ps1", 
+					"Install-PCoIPAgent.sh", 
+					"$using:linuxAgentARM", 
+					"$using:gaAgentARM",
+					"$using:agentARM", 
+					"$using:sumoAgentApplicationVM",
+					"$using:sumoConf",
+					"InstallPCoIPAgent.ps1.zip",
+					"$using:idleShutdownLinux"
+					)
+                Write-Host "Will upload these files: $new_agent_vm_files"
+				$acctKey = (Get-AzureRmStorageAccountKey -ResourceGroupName $using:RGName -AccountName $acct_name).Value[0]
+				$ctx = New-AzureStorageContext -StorageAccountName $acct_name -StorageAccountKey $acctKey
+				try {
+						Get-AzureStorageContainer -Name $container_name -Context $ctx -ErrorAction Stop
+				} Catch {
+						New-AzureStorageContainer -Name $container_name -Context $ctx -Permission "Blob"
+				}
+				Write-Host "Uploading files to private blob"
+				ForEach($file in $new_agent_vm_files) {
+						$filepath = Join-Path $using:LocalDLPath $file
+						try {
+							Get-AzureStorageBlob -Context $ctx -Container $container_name -Blob "remote-workstation\$file" -ErrorAction Stop
```

Comment:
Backslash or forward slash?

Reply:
Backlash works. They make it look like the windows file system. Although forward slash should also work

Reply:
For some anal-retentive reason I'd prefer forward slash here as I expect that's how most 'blob fake filesystems' are done. Or did you see it that way in a bunch of examples?